lib/resources.js

```diff
@@ -92,6 +92,17 @@ const FILTER_CONFIG = {
   }
 }
 
+const QUERY_STRING_QUERY_FIELDS = {
+  date: { field: 'dateStartYear' },
+  subject: { field: 'subjectLiteral' },
+  creator: { field: 'creatorLiteral' },
+  publisher: { field: 'publisherLiteral' },
+  title: { field: 'title' }
+}
```

Comment:
This defines what fields are allowed to be used as "query string query" fields (e.g. q=title:Romeo). This allows us to escape any colon that doesn't immediately follow an allowed field (e.g. q="2001: a space odyssey" becomes q="2001\: a space odyssey")